README.md

```diff
@@ -11,187 +11,32 @@ The Customers project focuses on solving authentic identification of humans who
 
 [FFIEC Bank Secrecy Act - Customer Identification Program](https://www.fdic.gov/regulations/examinations/bsa/ffiec_cip.pdf)
 
-Docs: [Project](https://github.com/moov-io/customers/tree/master/docs/) | [API Endpoints](https://moov-io.github.io/customers/api/) | [Admin API Endpoints](https://moov-io.github.io/customers/admin/)
+Docs: [docs](docs/README.md) | [API Endpoints](https://moov-io.github.io/customers/api/) | [Admin API Endpoints](https://moov-io.github.io/customers/admin/)
```

Comment:
I'd rather we use the GitHub pages hosted version. Is there a reason to switch?

Reply:
It seems that our generator should have it for all projects as well. Right? Will update templates as well.